packages/grid/src/vaadin-grid-selection-mixin.js

```diff
@@ -33,7 +33,10 @@ export const SelectionMixin = (superClass) =>
     }
 
     static get observers() {
-      return ['_itemIdPathChanged(itemIdPath)', '_selectedItemsChanged(selectedItems.*)'];
+      return [
+        '_updateSelectionForItemIdPathChange(itemIdPath)',
+        '_updateSelectionForSelectedItemsChange(selectedItems.*)'
```

Comment:
There is no need to rename `_selectedItemsChanged` - only the `_itemIdPathChanged` needs to be renamed.

Reply:
I've merged the two methods based on @tomivirkki 's comment below